pkg/controller/v1alpha2/experiment/experiment_controller.go

```diff
@@ -58,7 +62,30 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileExperiment{Client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	r := &ReconcileExperiment{
+		Client: mgr.GetClient(),
+		scheme: mgr.GetScheme(),
+	}
+	imp := viper.GetString(consts.ConfigExperimentSuggestionName)
```

Comment:
Just to check: Is there a better way to pass flags to the manager than using viper?

Reply:
I have no idea. Kubebuilder does not support a built-in way, thus I use viper here.

Reply:
Does it work for you?